File: easy/4020.py
"""
4020. Elevator Requests I

https://leetcode.com/problems/elevator-requests-i/

Biweekly Contest 189
"""


class Solution:
    def elevatorRequests(self, n: int, requests: list[int]) -> int:
        out: int = requests[0]

        for idx in range(1, len(requests)):
            out += abs(requests[idx] - requests[idx - 1])

        return out

File: easy/4020.rs
// 4020. Elevator Requests I
// https://leetcode.com/problems/elevator-requests-i/
// Biweekly Contest 189

impl Solution {
    pub fn elevator_requests(n: i32, requests: Vec<i32>) -> i32 {
        let mut out: i32 = requests[0];

        for idx in 1..requests.len() {
            out += (requests[idx] - requests[idx - 1]).abs();
        }

        out
    }
}
